src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -1,29 +1,30 @@
 package com.minecolonies.coremod.items;
 
 import net.minecraft.item.Item;
+import net.minecraftforge.fml.common.registry.IForgeRegistry;
 
 /**
  * Class handling the registering of the mod items.
+ *
+ * We disabled the following finals since we are neither able to mark the items as final, nor do we want to provide public accessors.
  */
+@SuppressWarnings({"squid:ClassVariableVisibilityCheck", "squid:S2444", "squid:S1444"})
 public final class ModItems
 {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make buildTool a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)